build.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
 set -e
 
+export DAPP_TEST_TIMESTAMP=1622483493
+export DAPP_TEST_NUMBER=12543537
 export DAPP_SOLC_VERSION=0.6.11
 export DAPP_SRC="contracts"
-export DAPP_BUILD_OPTIMIZE=1
+export DAPP_BUILD_OPTIMIZE=0
```

Comment:
If `optimize` is disabled, then I'm pretty sure `runs` does not matter, correct?